tests/test_google_search.py

```diff
@@ -60,6 +60,7 @@ def test_get_html(self):
 				search = GoogleSearch({"q": "Coffee", "location": "Austin,Texas"})
 				data = search.get_html()
 				self.assertGreater(len(data), 10)
+				self.assertIn("<html>", data)
 
```

Comment:
Let's double-check that `get_dict` still returns dictionary after calling `get_html`.

```suggestion

				data = search.get_dict()
				self.assertEqual(data["search_metadata"]["status"], "Success")
				self.assertIsNone(data.get("error"))
```